gradle/mkdocs.gradle

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+def gradleProject = project
+if (gradleProject == gradleProject.rootProject) {
+    apply plugin: 'org.hidetake.ssh'
+    apply plugin: 'ru.vyarus.mkdocs'
+}
+
+mkdocs {
+    sourcesDir = "$rootDir/documentation/src"
+    updateSiteUrl = false
+    strict = false
+    buildDir = 'build/mkdocs/documentation/generated'
+}
+
+remotes {
```

Comment:
What is this section supposed to be doing exactly?

Reply:
It is supposed to remote copy the generated HTML files to the webserver.